priv/templates/skf.gen.makefile/Makefile

```diff
@@ -77,11 +77,11 @@ set-remotes: require-DEVELOPMENT require-STAGING require-PRODUCTION
 
 ## Attach to IEx console on Heroku {REMOTE}
 heroku-console: require-REMOTE
-	@MIX_ENV=dev heroku run "iex -S mix" --remote ${REMOTE}
-	
+	heroku run "iex -S mix" --remote ${REMOTE}
```

Comment:
Haven't been following scaffolding part. Does the skafolder assume, that we will use OTP releases and docker based deployments on Heroku? If so the run "iex -S mix" command won't work.

Reply:
Skafolder doesn't assume it. I didn't still work on the release part. Why exactly would this command not work?

Reply:
If we use OTP releases on Heroku mix is not available.

Reply:
OK, that's a good point. Do we use releases on heroku on some project and which command do we use then?

Reply:
Telehealth and WFM. We use `./bin/telehealth start_iex` (after running bash).

Reply:
Thanx! Can we assume that all future projects will use OTP release on heroku? If so, I'll adapt this accordingly, but not in this PR.